WhlgPublicWebsite.BusinessLogic/Models/Questionnaire.cs

```diff
@@ -25,7 +25,13 @@ public record Questionnaire
 
     public EpcDetails EpcDetails { get; set; }
     public EpcConfirmation? EpcDetailsAreCorrect { get; set; }
-    public bool? IsLsoaProperty { get; set; }
+
+    /// <summary>
+    /// Whether the postcode is in IMD postcode deciles 1-2.
+    /// If so, the income band is not used to determine eligibility.
+    /// See <seealso cref="WhlgPublicWebsite.BusinessLogic.Services.EligiblePostcode.EligiblePostcodeService"/>
+    /// </summary>
```

Comment:
This comment is useful, should avoid future queries around income eligibility. Thank you!